### ISSUE TITLE
Android/ Weather-page text sections lack visual separation

### ISSUE THREAD

**Issue by helmines25** [fmidev/weather-app#998]
On Android's large font and display settings, wind speed, UV-index and air feels like- text sections (inside Today's weather- section) shift too close together and lack visual separation, making them difficult to distinguish.

Steps to reproduce:

- Enable larger display and font sizes in the Android device's settings
- Open the Weather view
- Observe how the text inside Todays weather section scales

**WCAG 1.4.4, AA**
_Ensure that all text can be resized up to 200% without the loss of content or functionality_

---

Screenshot for reference:
<img width="366" alt="Image" src="https://github.com/user-attachments/assets/fc2ccb1f-31da-4e53-983e-37e97168b7de" />
